public/sw.js

```diff
@@ -0,0 +1,56 @@
+var urlsToCache = [
+  '/',
+  'assets/css/main.css',
+  'assets/img/pattern_130.gif',
+  'assets/js/main.js',
+  'assets/vendor/bootstrap/bootstrap.js',
+  'assets/vendor/bootstrap/bootstrap.min.css',
+  'assets/vendor/prism/prism.js',
+  'assets/vendor/prism/prism.css',
+  'assets/vendor/bootstrap.modalform.js',
+  'assets/vendor/jquery-2.1.0.min.js',
+  'hoodie/client.js',
+  'index.html',
```

Comment:
if the service worker is always added in `index.html` there is no need to cache that (that will happen automatically). I think there is a mix of code here between my first and second commit, you can just forgot about my second commit altogether if we're just going to add it in the first page ;-).